syntax="proto2";
                                                                  
option java_package = "storageserver";                              
option java_outer_classname = "StorageServerProto";                 
                                                                  
package perfuse;                                                  
                                                                  
import "defines.proto";                                          

service StorageServer {                                             
    rpc GetChunkData(ChunkInfo) returns (ChunkData) {}                
    rpc PushChunkData(stream ChunkInfoData) returns (Error) {}
    rpc GetCapacity(Empty) returns (Capacity) {}
    rpc pingClient(NodeInfo) returns (Delay) {}
}
